src/main/java/net/dries007/tfc/api/types/Plant.java

```diff
@@ -276,6 +276,16 @@ public int getAgeForWorldgen(Random rand, float temp)
         return rand.nextInt(Math.max(1, Math.min(Math.round(2.5f + ((temp - minGrowthTemp) / minGrowthTemp)), 4)));
     }
 
+    public boolean canBePotted()
+    {
+        return (isCrossModel() && !(oreDictName.isPresent() && oreDictName.get().equals("reed"))) || plantType == PlantType.CACTUS || plantType == PlantType.CREEPING || plantType == PlantType.TALL_PLANT;
```

Comment:
Would rather have `(isCrossModel() && !(oreDictName.isPresent() && oreDictName.get().equals("reed")))` after the simple equality checks due to short circuiting.